packages/cli/src/tests/utils/actions.test.js

```diff
@@ -0,0 +1,39 @@
+const { expect } = require('chai');
+const { validateActions, ACTION_TYPES } = require('../../utils/actions');
+
+describe('actions utility', () => {
+  describe('validateActions', () => {
+    it('should validate correct action formats', () => {
+      const actions = ['trigger/test', 'create/item', 'search/find'];
```

Comment:
examples could be slightly better, trigger/new_item, create/create_item, search/find_item